src/components/header/header.tsx

```diff
@@ -101,17 +103,18 @@ const MenuLink = styled(Link)<ActiveProps>`
   height: 20px;
   position: relative;
 
-  color: ${(props) => (props.active ? "#60c2a4" : props.theme.fontColor)};
+  color: ${(props) =>
+    props.active ? lightGreen : props.theme.colors.fontColor};
   margin-right: 24px;
 
   &:hover {
-    color: #60c2a4;
+    color: ${lightGreen};
```

Comment:
Use purpose of color for naming instead of color name. Eg. `clMenuItemSelected`. Shall we use upper case for constants? Eg. CL_MENU_ITEM_SELECTED. We need prefix for color constant. I suggest `CL` or `COLOR`. We would need to use constants for other UI numbers, eg intentations.

Reply:
I don't see the point of naming constants by color name. We can just use hex then.

Reply:
The idea is to keep all website colors in `colors.ts `file to reuse it in multiple places like themes or just colors across different components.